src/metadata/metadata_raw_atomic.c

```diff
@@ -78,8 +79,8 @@ static int _raw_atomic_io_discard_do(struct ocf_cache *cache, void *context,
 		return req->error;
 	}
 
-	OCF_DEBUG_PARAM(cache, "Page to flushing = %u, count of pages = %u",
-			start_line, len);
+	OCF_DEBUG_PARAM(cache, "Page to flushing = %llu, count of pages = %u",
```

Comment:
It's `%llu` in OCL but `%lu` in posix. If we care about compiling ocf with debug flags with posix env, you can use `ENV_PRIu64` macro